MapboxNavigation/NavigationMapView.swift

```diff
@@ -625,7 +625,10 @@ open class NavigationMapView: MLNMapView, UIGestureRecognizerDelegate {
                 arrow.minimumZoomLevel = minimumZoomLevel
                 arrow.lineCap = NSExpression(forConstantValue: "butt")
                 arrow.lineJoin = NSExpression(forConstantValue: "round")
-                arrow.lineWidth = NSExpression(format: "mgl_interpolate:withCurveType:parameters:stops:($zoomLevel, 'linear', nil, %@)", MBRouteLineWidthByZoomLevel.multiplied(by: 0.70))
+				arrow.lineWidth = NSExpression(forMLNInterpolating: .zoomLevelVariable,
+											   curveType: .linear,
+											   parameters: nil,
+											   stops: NSExpression(forConstantValue: MLNRouteLineWidthByZoomLevel.multiplied(by: 0.7)))
```

Comment:
Please stick to the established indentation style.

```suggestion
                arrow.lineWidth = NSExpression(
                    forMLNInterpolating: .zoomLevelVariable,
                    curveType: .linear,
                    parameters: nil,
                    stops: NSExpression(
                        forConstantValue: MLNRouteLineWidthByZoomLevel.multiplied(by: 0.7)
                    )
                )
```